operator/config/default/kustomization.yaml

```diff
@@ -23,7 +23,7 @@ components:
   - ../crd
   - ../rbac
   # [WATCHER] To enable the watcher, uncomment all the sections with [WATCHER]
-  #- ../watcher
+  - ../watcher
```

Comment:
watcher should be disabled by default

Reply:
yes I am aware this for the reviewer to check

Reply:
anyways the Kustomize build has to be fixed